daemon/volumes.go

```diff
@@ -287,6 +290,27 @@ func (daemon *Daemon) verifyVolumesInfo(container *Container) error {
 	return container.ToDisk()
 }
 
+func (daemon *Daemon) migrateVolume(id, hostPath string) error {
```

Comment:
yeah, this is only called when there is a `/var/lib/docker/vfs/dir/VOLUME_ID` directory, which only happens using the local driver in versions < 1.7. After 1.7 all volumes are in `/var/lib/docker/volumes`.